__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -98,11 +99,32 @@ describe('<PropertyComponent />', () => {
         }
       }
 
-      const wrapper = shallow(<PropertyComponent propertyTemplate={template} />)
+      const wrapper = shallow(<PropertyComponent propertyTemplate={template}
+                               reduxPath={["http://id.loc.gov/ontologies/bibframe/note"]}/>)
       expect(wrapper.find('Connect(InputListLOC)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLiteral)').length).toEqual(0)
     })
   })
 
+  it('error if <PropertyComponent /> is missing reduxPath props', () => {
+    const template = {
```

Comment:
Does this test require its own `template` or can it use the `template` defined above? It'd help keep this test super focused on testing the `reduxPath` prop. No biggie either way, though.

Reply:
Would need to pull out the previous `template` into the enclosing `describe` block; I personally like the the template here to provide context but maybe this particular test should be in it's `describe` block.

Reply:
ok, no problem!